src/btree/bt_handle.c

```diff
@@ -939,21 +939,22 @@ __btree_page_sizes(WT_SESSION_IMPL *session)
           "B < %" PRIu32 "B)",
           btree->maxmempage_image, max);
 
-    /*
-     * Don't let pages grow large compared to the cache size or we can end
-     * up in a situation where nothing can be evicted.  Make sure at least
-     * 10 pages fit in cache when it is at the dirty trigger where threads
-     * stall.
-     *
-     * Take care getting the cache size: with a shared cache, it may not
-     * have been set.  Don't forget to update the API documentation if you
-     * alter the bounds for any of the parameters here.
-     */
+/*
+ * Don't let pages grow large compared to the cache size or we can end
+ * up in a situation where nothing can be evicted.  Make sure at least
+ * 10 pages fit in cache when it is at the dirty trigger where threads
+ * stall.
```

Comment:
I would suggest keeping this comment indented (as it was originally), otherwise our internal formatting testing may complain.

Reply:
Thanks for making the recent changes with s_all, one last change before we can get this merged @y123456yz.

I would suggest keeping this comment indented (as it was originally). However, the `#define WT_MIN_PAGES 10` is good where it is.